packages/Babylonian-Tests.package/BPPrintbuggerTest.class/instance/initExample1WithMethodWithMultipleProbes.st

```diff
@@ -0,0 +1,14 @@
+running
+initExample1WithMethodWithMultipleProbes
+	
+	example1 := (BPTestManuallyInstrumentedClass>>#methodWithMultipleProbes) bpExamples first.
+	[BPTestManuallyInstrumentedClass new methodWithMultipleProbes] bpTraceForExample: example1.
+
+	BPExamples value add: example1.
+	
+	"Force update notifications"
+	example1 isActive: true.
+	example1 changed: #exampleIsActive with: self.
```

Comment:
Why is that explicit `#changed:with:` call necessary?

Reply:
When examples are created, they are active by default. however, they are not added to BPExamples yet and therefore not watched by BPPrintbugger. I need at least one notification so the BPPrintbugger also knows what's up.
I cannot add the examples in `setUp` as they are neither nil (nil should not have dependents error) oder initialized without a method (UndefinedObject cannotUnderstand #reference). They have to be added in the test methods.
So I have to force one changed like isActive would do. I can either call it here explicitly (and delete line 10) or I can replace it with
```
example1 isActive: false.
example1 isActive: true.
```

Reply:
The example should not be active at this point in time. It should be sufficient to do:

```Smalltalk
  example1 := (BPTestManuallyInstrumentedClass>>methodWithMultipleProbes) bpExamples first.
  example1 isActive: true.
  "Activate that probes have trace values"
  example1 trace.
```

If anything else is needed there is a bug somewhere.

Reply:
Nope, doesn't work - BPExamples value activeExamples is empty in tests. also, example1 is created as active==true so that the second line will not change anything and not send the signal

Reply:
You probably also have to add https://github.com/hpi-swa-lab/babylonian-programming-smalltalk/pull/47#discussion_r649194184

Reply:
oh didnt see that! yeah youre right, now it works